src/lib/snark_worker_lib/prod.ml

```diff
@@ -38,24 +38,29 @@ module Inputs = struct
   end
 
   module Proof = Transaction_snark.Stable.V1
-  module Statement = Transaction_snark.Statement
+  module Statement = Transaction_snark.Statement.Stable.V1
 
   module Public_key = struct
     include Public_key.Compressed
 
     let arg_type = Cli_lib.Arg_type.public_key_compressed
   end
 
-  module Transaction = Coda_base.Transaction
-  module Sparse_ledger = Coda_base.Sparse_ledger
-  module Pending_coinbase = Coda_base.Pending_coinbase
+  module Transaction = Coda_base.Transaction.Stable.Latest
```

Comment:
If these need to be `bin_io`, and they're used as functor inputs, the particular choice creates a new version. I'd say the inputs need to be versioned, like `Inputs1`, etc.